app/test/integration-test/src/test/java/io/syndesis/test/itest/mail/SendMail_IT.java

```diff
@@ -129,7 +129,7 @@ public void testSendMailError(@CitrusResource TestCaseRunner runner) {
 
         runner.variable("first_name", "Joanne");
         runner.variable("company", "Red Hat");
-        runner.variable("email", "joanne@syndesis.org");
+        runner.variable("email", "joan");
```

Comment:
Can you please revert this particular change. It should be addressed and properly fixed in #8811